### PR TITLE
A post about our ghc+stackage update

### PR DESCRIPTION
**Pull request by telser**
Feel free to be harsh, I was very unsure of target audience, so how much depth to go into on "basics" like what stackage/hlint/ghc/etc are. And other stuff like length overall, use numbers like 1 or one, and so on...

### CODE REVIEW

**Comment by telser** [comment]
PS it has been a very long time since I looked at AP guidelines, and was always more familiar with, the less relevant, MLA.

**Review by tfausak — COMMENTED** [review]
Looks great! There are two things I am interested in that would require doing some extra work to get: Compile times before/after and runtime resource usage before/after.

**Comment by tillydray** [comment]
It would be good to put this through a markdown linter before merging it.

File: _posts/2018-06-08-upgrading-ghc.markdown [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Haskell In Production: A GHC Upgrade Success Story
+
+Version upgrade nightmares are so common that they become almost expected. Update the compiler one patch level, or a library version, and the work spirals into a much larger project.
```
Comment (tfausak) [inline _posts/2018-06-08-upgrading-ghc.markdown]: I feel like "become" isn't quite the right word here. What about "are" or "have become"?

Comment (telser) [reply _posts/2018-06-08-upgrading-ghc.markdown]: :+1: have become

**Review by tfausak — APPROVED** [review]

**Pull request merged**